src/types/types.ts

```diff
@@ -0,0 +1,6 @@
+export type TtokenData =  {
```

Comment:
typo?

Reply:
Not a type. It was intentional go put the T there

Reply:
Got it, but I'd suggest in this case using such naming: `TTokenData`